packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
-# Customize inbound failsafe rules
-# https://docs.projectcalico.org/v3.5/getting-started/bare-metal/policy/failsafe
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
+
+# Host endpoints
+# ==============
+---
+${controller_host_endpoints}
+---
+${worker_host_endpoints}
+
+# Firewall policy
+# ===============
+---
+# SSH access
```

Comment:
Hmm, following https://kubernetes.io/docs/concepts/configuration/overview/#general-configuration-tips, maybe we could put those comments in annotations? That seems like a good practice.

Reply:
I'm not sure what this means. Can you provide a sample YAML snippet?

Reply:
I was thinking about something like:
```diff
--- a/packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl
+++ b/packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl
@@ -53,11 +53,12 @@ spec:
     destination:
       ports: [6443]
 ---
-# Traffic between cluster nodes
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
   name: inter-node-traffic
+  annotations:
+    description: 'Traffic between cluster nodes'
 spec:
   selector: has(host-endpoint)
   preDNAT: true
```

BTW, this comment/description duplicates a bit with resource name. Maybe we could remove it completely or make it more descriptive? Eg. `Allow all traffic across cluster internal CIDRs`

Reply:
We can add annotations to our rules. I would do this in a new PR as we currently don't use annotations as descriptions for k8s objects and in any case this is out of scope here. Feel free to open a new PR and put me as a reviewer, and thanks for the suggestion.